### PR TITLE
[shopsys] UPGRADE-v7.x.x tweak because of conflict with symfony/monolog-bundle:3.4.0

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| hotfix of problem with BC of project using newest monolog-bundle version 3.4.0
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues|  
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: docs/upgrade/UPGRADE-v7.0.1.md [hunk 1]
```diff
@@ -19,5 +19,8 @@ There you can find links to upgrade notes for other versions too.
     - if you haven't extended any of these methods and haven't called `new MessageData(...)` in your code, you should be protected by the upgrade alone
     - if you have customized e-mailing on your project (eg. by implementing Twig templates for mail content), you should check your code to avoid double escaping (eg. execute `htmlspecialchars_decode($value, ENT_QUOTES)` before passing the variables replacements to you implementation)
 
+### Configuration
+- do not update `symfony/monolog-bundle` to the higher version than `3.3.1` ([#1148](https://github.com/shopsys/shopsys/pull/1148))
```
Comment (grossmannmartin) [inline docs/upgrade/UPGRADE-v7.0.1.md]: Please make it consistent with linked PR 
```suggestion
- do not update `symfony/monolog-bundle` to the version `3.4.0` and higher ([#1148](https://github.com/shopsys/shopsys/pull/1148))
```

Comment (grossmannmartin) [reply docs/upgrade/UPGRADE-v7.0.1.md]: And can you link the issue from monolog to be clear why it's forbidden?

File: docs/upgrade/UPGRADE-v7.0.1.md [hunk 2]
```diff
@@ -19,5 +19,8 @@ There you can find links to upgrade notes for other versions too.
     - if you haven't extended any of these methods and haven't called `new MessageData(...)` in your code, you should be protected by the upgrade alone
     - if you have customized e-mailing on your project (eg. by implementing Twig templates for mail content), you should check your code to avoid double escaping (eg. execute `htmlspecialchars_decode($value, ENT_QUOTES)` before passing the variables replacements to you implementation)
 
+### Configuration
+- do not update `symfony/monolog-bundle` to the version than `3.4.0` and higher before [symfony/monolog-bundle#309](https://github.com/symfony/monolog-bundle/issues/309) is resolved ([#1148](https://github.com/shopsys/shopsys/pull/1148))
```
Comment (grossmannmartin) [inline docs/upgrade/UPGRADE-v7.0.1.md]:
```suggestion
- do not update `symfony/monolog-bundle` to the version `3.4.0` and higher before [symfony/monolog-bundle#309](https://github.com/symfony/monolog-bundle/issues/309) is resolved ([#1148](https://github.com/shopsys/shopsys/pull/1148))
```

**Comment by PetrHeinz** [comment]
I think I've prepared the proper fix of the configuration in #1154. And because the configuration is in fact in the `project-base` part, we can copy-paste the upgrade note:

```markdown
- unset the incompatible `excluded_404s` configuration from monolog handlers that don't use the `fingers_crossed` type ([#1154](https://github.com/shopsys/shopsys/pull/1154))
    - in `app/config/packages/dev/monolog.yml`:
    ```diff
        monolog:
           handlers:
               main:
                   # change "fingers_crossed" handler to "group" that works as a passthrough to "nested"
                   type: group
                   members: [ nested ]
    +              excluded_404s: false
    ```
    - in `app/config/packages/test/monolog.yml`:
    ```diff
        monolog:
            handlers:
                main:
                    type: "null"
    +               excluded_404s: false
```

**Pull request merged**